src/dsp_tools/commands/ingest_xmlupload/create_resources/apply_ingest_id.py

```diff
@@ -58,13 +58,7 @@ def replace_filepath_with_internal_filename(
             continue
         img_path_str = elem.text
         if img_path_str not in orig_path_2_asset_id:
-            img_path = Path(elem.text)
-            img_path_str = str(img_path.relative_to(Path.cwd()) if img_path.is_absolute() else img_path)
-            if img_path_str not in orig_path_2_asset_id:
-                img_path_str = str(img_path.with_suffix(img_path.suffix.lower()))
-            if img_path_str not in orig_path_2_asset_id:
-                img_path_str = str(img_path.with_suffix(img_path.suffix.upper()))
-
+            img_path_str = "/" + img_path_str
```

Comment:
This is the fix for https://linear.app/dasch/issue/DEV-4166/ingest-omits-leading-slash-in-mapping-csv. Has anyone an idea how to test this? I believe it's untestable.

Reply:
I would say it's ok to not test this